src/requirements/Payment/components/RegisterVaultForm/RegisterVaultForm.tsx

```diff
@@ -98,13 +125,14 @@ const RegisterVaultForm = ({ isDisabled }: Props): JSX.Element => {
 
         <InputGroup>
           <NumberInput
-            isDisabled={isDisabled}
+            isDisabled={isDisabled || !token || !chain}
```

Comment:
I'd maybe disable this input until `tokenData` is not defined, what do you think?